src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -41,6 +41,8 @@ import RelatedTopics from '#containers/RelatedTopics';
 import NielsenAnalytics from '#containers/NielsenAnalytics';
 import ScrollablePromo from '#components/ScrollablePromo';
 import CpsRecommendations from '#containers/CpsRecommendations';
+import EmbedImages from '../../components/Embeds/EmbedImages';
+import EmbedHtml from '../../components/Embeds/EmbedHtml';
```

Comment:
If I remember rightly we also decided to add this to the MediaArticle Pages. (With the undertstanding that it might look bad on dark mode pages but that should be for editorial to decide).

Reply:
Ahh yeah forgot, I shall add them thanks